src/test/java/redis/clients/jedis/JedisPooledClientSideCacheTest.java

```diff
@@ -0,0 +1,100 @@
+package redis.clients.jedis;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.function.Supplier;
+import org.apache.commons.pool2.impl.GenericObjectPoolConfig;
+import org.hamcrest.Matchers;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class JedisPooledClientSideCacheTest {
+
+  protected static final HostAndPort hnp = HostAndPorts.getRedisServers().get(1);
+
+  protected Jedis control;
+
+  @Before
+  public void setUp() throws Exception {
+    control = new Jedis(hnp, DefaultJedisClientConfig.builder().password("foobared").build());
```

Comment:
foobared is used enough times that it might as well be `static final` internal to the tests only